pkg/scheduler/framework/plugins/defaultpreemption/default_preemption.go

```diff
@@ -97,8 +97,9 @@ func (pl *DefaultPreemption) PostFilter(ctx context.Context, state *framework.Cy
 	}
 
 	result, status := pe.Preempt(ctx, pod, m)
-	if status.Message() != "" {
-		return result, framework.NewStatus(status.Code(), "preemption: "+status.Message())
+	msg := status.Message()
```

Comment:
shouldn't we change this message not to include a period?

Reply:
Because currently default preemption plugin will contain the status as below
https://github.com/kubernetes/kubernetes/blob/91344b4008f6c5dfce82541b0c96cb8670b8350c/pkg/scheduler/framework/preemption/preemption.go#L183-L186

And output the message like `Filter` status which contains the **period** https://github.com/kubernetes/kubernetes/blob/91344b4008f6c5dfce82541b0c96cb8670b8350c/pkg/scheduler/framework/types.go#L333-L350

I'm ok to leave it as it is, just a little OCD. 🥲